atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -649,3 +649,10 @@ def run(self):
         self.workflow.default_image_build_method = default_image_build_method
         self.workflow.builder.tasker.build_method = (source_image_build_method or
                                                      default_image_build_method)
+
+        # set source registry and organization
+        if self.workflow.builder.dockerfile_images is not None:
```

Comment:
dockerfile_images are set with set_df_path in InsideBuilder (builder.py) WHEN Dockerfile exists (not-flatpak)
so reactor_config checks for None, and sets source_reg & org in that case

in flatpak_create_dockerfile the Dockerile is created so in that case we call set_df path there which sets dockerfile_images, and then set source_registry & org

Reply:
ic, so basically, only plugins that run before `flatpak_create_dockerfile` have to check for `None` in case build is a flatpak build and Dockerfile does not exist?

Reply:
exactly